snapshot_deploy.sh

```diff
@@ -47,6 +47,6 @@ echo "[Info] Sign package and deploy to staging area"
 project_version=$($THIS_DIR/scripts/get_project_info_from_pom.py $THIS_DIR/pom.xml version)
 $THIS_DIR/scripts/update_project_version.py public_pom.xml $project_version > generated_public_pom.xml
 
-mvn clean deploy ${MVN_OPTIONS[@]} -Dsnapshot-deploy
+mvn clean deploy ${MVN_OPTIONS[@]} -Dsnapshot-deploy -DskipTests=true
```

Comment:
I think we should run the tests to make sure everything works?

Reply:
Since we are validating all the tests with RTs, I thought we can skip testing with Jenkins. Wdyt ?